piksi_tools/action_logger.py

```diff
@@ -254,9 +254,9 @@ def main():
           link.add_callback(sl.printer, SBP_MSG_PRINT_DEP)
```

Comment:
We can and should apply the Forwarder wherever the callbacks can block or take time to run.  There is no real gain where they are quick, or just post something to run on the UI thread.